WebsiteGreetz/src/test/resources/MySuite.xml

```diff
@@ -0,0 +1,15 @@
+<!DOCTYPE suite SYSTEM "https://testng.org/testng-1.0.dtd" >
+
+<suite name="mySuite" verbose="1" >
+    <test name="myTest">
+        <classes>
+            <class name="WebsiteTest">
+                <methods>
+                  <include name="addAndCheckFavorite()"/>
```

Comment:
Are you sure this sute file works? Parenthesis should not be there

Reply:
Yes, it works. I will check it once again.